src/Templates/src/Microsoft.Maui.Templates.csproj

```diff
@@ -58,27 +58,16 @@
     <BeforePack>_UpdateTemplateVersions</BeforePack>
   </PropertyGroup>
 
```

Comment:
Were any changes to this file deliberate? If not, please undo the changes to this file.